### ISSUE TITLE
Replace icons with something retina compatible / scalable

### ISSUE THREAD

**Issue by leppert** [harvard-lil/h2o#437]
We'll probably want SVGs. 
![image](https://user-images.githubusercontent.com/26643/42049455-08fb002c-7ad4-11e8-9feb-8839025e35a3.png)

**Comment by rebeccacremona** [comment]
A good, reasonably recent discussion about options for implementing accessible icons: https://www.24a11y.com/2017/svg-icon-fonts-accessibility-case-study/. Links to more resources at the bottom. It's clear how to do this well for everybody except users with low vision, who often prefer to adjust the colors of their screens: icons don't always come along for the ride. I'm not sure what the best technique is in 2018.

**Comment by leppert** [comment]
Design files, possibly containing vector icons:
https://www.dropbox.com/s/iv0uhn7o8ie0h67/H2O-final-package.zip?dl=0

**Comment by cgruppioni** [comment]
Color ideas from #434 

- [ ] Change the preview button icon ( currently an ellipsis ) to something more relevant
- [ ] Currently the Publish button is yellow (like the draft mode screen), and the Revise button is blue. Should these colors be swapped, since they each trigger the user to go to the other space? (i.e. a blue Publish button takes you to the blue Publish screen).
- [ ] Clone and Add Resource sidebar buttons are green, but others (ex. Add Section) are not. Feels like these colors should be consistent.

**Comment by rebeccacremona** [comment]
Done in https://github.com/harvard-lil/h2o/pull/978
